interface/modules/custom_modules/oe-module-dorn/public/orders.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once "../../../../globals.php";
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Csrf\CsrfUtils;
+use OpenEMR\Common\Twig\TwigContainer;
+use OpenEMR\Modules\Dorn\ConnectorApi;
+use OpenEMR\Core\Header;
+//this is needed along with setupHeader() to get the pop up to appear
+
+$tab = "orders";
+$pageTitle = xlt("DORN Orders");
+if (!AclMain::aclCheckCore('admin', 'users')) {
+    echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl($pageTitle)]);
+    exit;
+}
+$primaryInfos = ConnectorApi::getPrimaryInfos('');
+if (!empty($_POST)) {
+    if (isset($_POST['SubmitButton'])) {
+    //check if form was submitted
+        $datas = ConnectorApi::searchOrderStatus($_POST['form_orderNumber'], $_POST['form_primaryId'], $_POST['form_startDateTime'], $_POST['form_endDateTime']);
+        if ($datas == null) {
+            $datas = [];
+        }
+    }
+}
+?>
+<html lang="">
+<head>
+        <?php Header::setupHeader(); ?>
+    <title> <?php echo $pageTitle; ?>  </title>
+</head>
+<script>
+    $(function () {
+        $('.datepicker').datetimepicker({
+            <?php $datetimepicker_timepicker = false; ?>
+            <?php $datetimepicker_showseconds = false; ?>
+            <?php $datetimepicker_formatInput = true; ?>
```

Comment:
change above to false since functionality for internationalization of dates is not on this page

Reply:
I commented this wasn't done as unsure what Brad needs for dates here.